app/src/pages/playground/Playground.tsx

```diff
@@ -267,9 +271,15 @@ function PlaygroundContent() {
               >
                 {/* No padding on the right of the accordion item, it is handled by the stable scrollbar gutter */}
                 <View height="100%" paddingY="size-200" paddingStart="size-200">
-                  <Flex direction="row" gap="size-200">
+                  <Flex direction="row" gap="size-200" maxWidth="100%">
                     {instances.map((instance) => (
-                      <View key={instance.id} flex="1 1 0px">
+                      <View
+                        key={instance.id}
+                        // This width accomodates the model config button min-width, as well as chat message accordion
+                        // header contents such as the chat message mode radio group for AI messages
+                        minWidth="475px"
```

Comment:
small nit: pull this out into a constant so it can be referenced easier / reused if it needs to be updated etc.